libs/parser-sql/src/postgresParser.ts

```
@@ -0,0 +1,1652 @@
+const Enum = createToken({name: 'Enum', pattern: /\bENUM\b/i, longer_alt: Identifier})
+const Except = createToken({name: 'Except', pattern: /\bEXCEPT\b/i, longer_alt: Identifier})
+const Exists = createToken({name: 'Exists', pattern: /\bEXISTS\b/i, longer_alt: Identifier})
+const Extension = createToken({name: 'Extension', pattern: /\bEXTENSION\b/i, longer_alt: Identifier})
+const False = createToken({name: 'False', pattern: /\bFALSE\b/i, longer_alt: Identifier})
+const Fetch = createToken({name: 'Fetch', pattern: /\bFETCH\b/i, longer_alt: Identifier})
+const Filter = createToken({name: 'Filter', pattern: /\bFILTER\b/i, longer_alt: Identifier})
+const First = createToken({name: 'First', pattern: /\bFIRST\b/i, longer_alt: Identifier})
+const ForeignKey = createToken({name: 'ForeignKey', pattern: /\bFOREIGN\s+KEY\b/i})
+const From = createToken({name: 'From', pattern: /\bFROM\b/i, longer_alt: Identifier})
+const Full = createToken({name: 'Full', pattern: /\bFULL\b/i, longer_alt: Identifier})
+const Function = createToken({name: 'Function', pattern: /\bFUNCTION\b/i, longer_alt: Identifier})
```

Biome

[error] Do not shadow the global "Function" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)